tests/rules/pds3/test_coiss_xxxx.py

```
@@ -1,15 +1,16 @@
-import pytest
 import os
 
-import pdsfile.pds3file as pds3file
+import pytest
 
+import pdsfile.pds3file as pds3file
```

Ruff (0.16.1)

[warning] Use `from pdsfile import pds3file` in lieu of alias

Replace with `from pdsfile import pds3file`

(PLR0402)